client/app/states/login/login.html

```diff
@@ -1,4 +1,33 @@
-<div class="login-pf">
+<div class="login-pf" ng-if="vm.authenticationInfo && vm.authenticationInfo.oidc_enabled">
```

Comment:
looks like the only differences are inside the `<form` element?

If so, we should probably move the ng-if there and not repeat the logos...

Reply:
this also helped eliminate the period of time I had the screen white, before the ng-if expressions were valid. 👍